### PR TITLE
2063 by Inlead: Export to feature.

### PR DESCRIPTION
**Pull request by guddo**
http://platform.dandigbib.org/issues/2063

### CODE REVIEW

File: modules/ding_event/ding_event.features.field_instance.inc [hunk 1]
```diff
@@ -651,23 +647,22 @@ function ding_event_field_default_field_instances() {
       ),
     ),
     'entity_type' => 'node',
-    'fences_wrapper' => 'div',
     'field_name' => 'field_ding_event_price',
     'label' => 'Price',
-    'required' => 0,
+    'required' => 1,
```
Comment (kasperg) [inline modules/ding_event/ding_event.features.field_instance.inc]: This forces events to have a price. Is that intentional?

File: modules/ding_event/ding_event.features.field_instance.inc [hunk 2]
```diff
@@ -651,23 +647,22 @@ function ding_event_field_default_field_instances() {
...
-      'settings' => array(),
-      'type' => 'number',
-      'weight' => 5,
+      'active' => 1,
+      'module' => 'text',
+      'settings' => array(
+        'size' => 60,
+      ),
+      'type' => 'text_textfield',
+      'weight' => 8,
```
Comment (kasperg) [inline modules/ding_event/ding_event.features.field_instance.inc]: This changes the price widget from a number field to a textfield. Is that intentional?

File: modules/ding_event/ding_event.features.field_instance.inc [hunk 3]
```diff
@@ -617,31 +617,27 @@ function ding_event_field_default_field_instances() {
...
-        'weight' => 8,
+        'label' => 'above',
+        'module' => 'text',
+        'settings' => array(),
+        'type' => 'text_default',
+        'weight' => 3,
       ),
       'search_result' => array(
-        'label' => 'hidden',
+        'label' => 'above',
```
Comment (kasperg) [inline modules/ding_event/ding_event.features.field_instance.inc]: Is it intentional that we now want to show the label for the price field?

File: modules/ding_event/ding_event.features.field_base.inc [hunk 4]
```diff
@@ -346,15 +346,27 @@ function ding_event_field_default_field_bases() {
...
+      ),
+    ),
+    'indexes' => array(
+      'format' => array(
+        0 => 'format',
+      ),
+    ),
     'locked' => 0,
-    'module' => 'number',
+    'module' => 'text',
```
Comment (kasperg) [inline modules/ding_event/ding_event.features.field_base.inc]: This changes the price field from a number to a textfield. Is that intentional?

File: modules/ding_event/ding_event.features.field_instance.inc (hunk 3 above)
Comment (guddo) [inline modules/ding_event/ding_event.features.field_instance.inc]: Yes, this is a request from the business.

File: modules/ding_event/ding_event.features.field_instance.inc (hunk 1 above)
Comment (guddo) [inline modules/ding_event/ding_event.features.field_instance.inc]: Yes, this is a request from the business.

File: modules/ding_event/ding_event.features.field_instance.inc (hunk 2 above)
Comment (guddo) [inline modules/ding_event/ding_event.features.field_instance.inc]: Yes, this is a request from the business.

File: modules/ding_event/ding_event.features.field_base.inc (hunk 4 above)
Comment (guddo) [inline modules/ding_event/ding_event.features.field_base.inc]: Yes, this is a request from the business.

**Comment by guddo** [comment]
This should correspond to the hook_update() changes made in https://github.com/ding2/ding2/pull/455.

**Comment by kasperg** [comment]
Thanks for the clarification @guddo.

Reviewed and approved.

**Pull request merged**